maildir2gmail.py

```diff
@@ -58,7 +58,8 @@ def append(self, filename):
         timestamp = parsedate(message['date'])
         if not timestamp:
             log('Skipping "%s" - no date' % os.path.basename(filename))
-            return
+            timestamp = 0
```

Comment:
Well, that will change the overall behaviour of the script. Can you make this an option please?